pm/recipient.go

```diff
@@ -443,6 +443,6 @@ func (r *recipient) redeemManager() {
 	}
 }
 
```

Comment:
Add method comment here?

Reply:
Added in https://github.com/livepeer/go-livepeer/pull/956/commits/789195926a3d4628b3bc3e6acec20f4b78035e68